src/test/java/bio/terra/service/auth/iam/IamServiceTest.java

```diff
@@ -59,6 +60,23 @@ void setup() {
             mock(GoogleCredentialsService.class));
   }
 
+  @Test
+  void testRetrieveAuthDomain() throws InterruptedException {
+    when(iamProvider.retrieveAuthDomain(TEST_USER, IamResourceType.DATASNAPSHOT, ID))
+        .thenReturn(AUTH_DOMAIN);
+
+    List<String> result =
+        iamService.retrieveAuthDomain(TEST_USER, IamResourceType.DATASNAPSHOT, ID);
+    verify(iamProvider).retrieveAuthDomain(TEST_USER, IamResourceType.DATASNAPSHOT, ID);
```

Comment:
This `verify` is unnecessary as this test uses strict stubbing, so we know the stub above was called at least once. Also the fact that it generates a result which is asserted also means that we know it called the stub.